Index: 2008_0506_RT73_Linux_STA_Drv1.1.0.1/Module/rtmp_def.h
=============================================================
--- Module/rtmp_def.h_orig	2008-05-06 09:56:44.000000000 +0430
+++ Module/rtmp_def.h	2012-06-09 15:04:45.000000000 +0430
@@ -860,6 +860,47 @@
  {USB_DEVICE(0x07d1,0x3c03)}, /* D-Link */  \
  {USB_DEVICE(0x13b1,0x0023)}, /* Linksys */  \
  {USB_DEVICE(0x13b1,0x0020)}, /* Linksys WUS54GC */ \
+ {USB_DEVICE(0x0411,0x00f4)}, /* BUFFALO */      \
+ {USB_DEVICE(0x0411,0x0116)}, /* BUFFALO */      \
+ {USB_DEVICE(0x0411,0x0119)}, /* BUFFALO */      \
+ {USB_DEVICE(0x0471,0x200a)}, /* Philips */      \
+ {USB_DEVICE(0x04bb,0x093d)}, /* I-O */      \
+ {USB_DEVICE(0x04e8,0x4471)}, /* Samsung */      \
+ {USB_DEVICE(0x050d,0x7050)}, /* Belkin */      \
+ {USB_DEVICE(0x050d,0x905c)}, /* Belkin */      \
+ {USB_DEVICE(0x06f8,0xe002)}, /* Guillemot */      \
+ {USB_DEVICE(0x06f8,0xe010)}, /* Guillemot */      \
+ {USB_DEVICE(0x06f8,0xe020)}, /* Guillemot */      \
+ {USB_DEVICE(0x07aa,0x002e)}, /* Corega */      \
+ {USB_DEVICE(0x07b8,0xb21b)}, /* AboCom */      \
+ {USB_DEVICE(0x07b8,0xb21c)}, /* AboCom */      \
+ {USB_DEVICE(0x07b8,0xb21e)}, /* AboCom */      \
+ {USB_DEVICE(0x07b8,0xb21f)}, /* AboCom */      \
+ {USB_DEVICE(0x07d1,0x3c04)}, /* D-Link */      \
+ {USB_DEVICE(0x07d1,0x3c06)}, /* D-Link */      \
+ {USB_DEVICE(0x07d1,0x3c07)}, /* D-Link */      \
+ {USB_DEVICE(0x0812,0x3101)}, /* Unknown */      \
+ {USB_DEVICE(0x08dd,0x0120)}, /* Billionton */      \
+ {USB_DEVICE(0x0cde,0x001c)}, /* Z-Com */      \
+ {USB_DEVICE(0x0db0,0x4600)}, /* MSI */      \
+ {USB_DEVICE(0x0db0,0xa874)}, /* MSI */      \
+ {USB_DEVICE(0x0df6,0x0024)}, /* Sitecom */      \
+ {USB_DEVICE(0x0df6,0x0027)}, /* Sitecom */      \
+ {USB_DEVICE(0x0df6,0x002f)}, /* Sitecom */      \
+ {USB_DEVICE(0x13b1,0x0028)}, /* Linksys */      \
+ {USB_DEVICE(0x15a9,0x0004)}, /* Gemtek */      \
+ {USB_DEVICE(0x1740,0x3701)}, /* Senao */      \
+ {USB_DEVICE(0x1740,0x7100)}, /* Senao */      \
+ {USB_DEVICE(0x18c5,0x0002)}, /* AMIT */      \
+ {USB_DEVICE(0x1b75,0x7318)}, /* Ovislink */      \
+ {USB_DEVICE(0x2019,0xab01)}, /* PLANEX */      \
+ {USB_DEVICE(0x2019,0xab50)}, /* PLANEX */      \
+ {USB_DEVICE(0x6933,0x5001)}, /* Unknown */      \
+ {USB_DEVICE(0x7167,0x3840)}, /* Unknown */      \
+ {USB_DEVICE(0x0411,0x00d9)}, /* BUFFALO */      \ 
+ {USB_DEVICE(0x0411,0x0137)}, /* BUFFALO */      \
+ {USB_DEVICE(0x07b8,0xab01)}, /* AboCom */      \
+ {USB_DEVICE(0x07b8,0x3415)}, /* AboCom */      \ 
  {}} /* end marker */
 
 #endif	// __RTMP_DEF_H__
