diff -Naur a/data/keymap.xml b/data/keymap.xml
--- a/data/keymap.xml	2012-11-29 14:34:42.000000000 +0100
+++ b/data/keymap.xml	2012-11-30 19:47:46.000000000 +0100
@@ -208,6 +208,7 @@
 
 	<map context="InfobarSubserviceSelectionActions">
 		<key id="KEY_GREEN" mapto="subserviceSelection" flags="b" />
+		<key id="KEY_YELLOW" mapto="failsafe" flags="b" />
 	</map>
 
 	<map context="InfobarRedButtonActions">		
diff -Naur a/lib/python/Screens/InfoBarGenerics.py b/lib/python/Screens/InfoBarGenerics.py
--- a/lib/python/Screens/InfoBarGenerics.py	2013-07-31 16:13:49.000000000 +0430
+++ b/lib/python/Screens/InfoBarGenerics.py	2013-07-31 16:56:03.152311330 +0430
@@ -2166,6 +2166,7 @@
 		self["SubserviceSelectionAction"] = HelpableActionMap(self, "InfobarSubserviceSelectionActions",
 			{
 				"subserviceSelection": (self.subserviceSelection, _("Subservice list...")),
+				"failsafe": (self.failsafe, _("failsafe...")),
 			})
 
 		self["SubserviceQuickzapAction"] = HelpableActionMap(self, "InfobarSubserviceQuickzapActions",
@@ -2183,6 +2184,12 @@
 
 		self.bsel = None
 
+	def failsafe(self):
+		try:
+			open("/proc/failsafe", "w").write("0")
+		except Exception, e:
+			print e
+
 	def __removeNotifications(self):
 		self.session.nav.event.remove(self.checkSubservicesAvail)
 
